apps/frontend/src/app/pages/company/build-training-data/build-training-data.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="container mx-auto min-h-screen px-3 sm:px-0">
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/71036d45-cc27-4c0f-aa45-46f1284a1971)

The responsiveness is a little off on mobile